### ISSUE TITLE
Error open No such file run the ./ChosunTruck [OpenCV error message]

### ISSUE THREAD

**Issue by mrasyidaqmar** [bethesirius/ChosunTruck#35]
Hi, basically this error is a continuation of error issue #17 posted by @gotope. 
Previously, I have successfully ran the ChosunTruck on Windows.
Now on Ubuntu 16.04, I had the same problem as @gotope's issue #17 , and ended up here:
~/ChosunTruck/linux$ sudo ./ChosunTruck 
OpenCV Error: Assertion failed (0 <= roi.x && 0 <= roi.width && roi.x + roi.width <= m.cols && 0 <= roi.y && 0 <= roi.height && roi.y + roi.height <= m.rows) in Mat, file /home/aqmar/OpenCV/modules/core/src/matrix.cpp, line 508
terminate called after throwing an instance of 'cv::Exception'
  what():  /home/aqmar/OpenCV/modules/core/src/matrix.cpp:508: error: (-215) 0 <= roi.x && 0 <= roi.width && roi.x + roi.width <= m.cols && 0 <= roi.y && 0 <= roi.height && roi.y + roi.height <= m.rows in function Mat

My suspicion right now is in the ETS2 windowed mode size is not appropriate.  (I have put the window in the right bottom though).
This is the screenshot:
![chosuntruck-opencv-error-snapshot](https://cloud.githubusercontent.com/assets/1209717/24130411/aad2e6a6-0e2b-11e7-9110-30e600068d22.png)

But I am not sure how to fix this yet. I it possible that my monitor resolution has a role on the error?

**Comment by chi3236** [comment]
Yeah. The monitor resolution in Linux should be 1920x1080, because of the limitation of getting screen method. Please check your monitor resolution and set it to 1920x1080, if not set.
